protocol/x/ratelimit/keeper/keeper.go

```diff
@@ -77,6 +77,15 @@ func (k Keeper) ProcessWithdrawal(
 			)
 		}
 
+		metrics.EmitTelemetryWithLabelsForExecMode(
+			ctx,
+			[]sdk.ExecMode{sdk.ExecModeFinalize},
+			metrics.IncrCounterWithLabels,
```

Comment:
Instead of a `IncrCounter` here should we use `AddSample` instead? Thinking about how to easily graph this in Datadog.

We are interested in amount withdrawn with a certain time window. Similarly we have this DD [chart](https://app.datadoghq.com/dashboard/fbc-gkh-ywa/v4-clob?fromUser=true&fullscreen_end_ts=1711486952418&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=edit&fullscreen_start_ts=1710882152418&fullscreen_widget=3228782431089520&refresh_mode=sliding&view=spans&from_ts=1710882149774&to_ts=1711486949774&live=true) for amount liquidated in some window